### DISCUSSION TITLE
【交易】收购各种录像带，底片。价格好商量！！

### DISCUSSION THREAD

**Discussion by glad-C** [share121/inter-knot#1462] [其他]
![web1](https://act-upload.mihoyo.com/nap-obc-indep/2024/05/21/284550490/ac51f7f15784b399abaa3afc0cc533e6_984961226078874111.png)
![web2](https://act-upload.mihoyo.com/nap-obc-indep/2024/05/20/284550490/db69cda0defca40314c0676e5c194fe2_5425955207018759955.png）
![web3]（https://act-upload.mihoyo.com/nap-obc-indep/2024/07/06/284550490/7f2333755e3bee83a6f9fa5e74b6bd41_5317861006780870817.png）
![web4]（https://act-upload.mihoyo.com/nap-obc-indep/2024/05/22/284550490/8ff8462918776b83f8d1482dbf8f1a85_5051556221623384631.png）

**Comment by Dave-12138** [comment]
笑点解析：楼主把半角括号()打成中文全角括号（）了

**Comment by capacitor1** [comment]
残缺不全的markdown🤔

**Comment by su2569** [comment]
6
